metastats-domain/src/test/groovy/life/qbic/metastats/filter/FilterExperimentDataSpecification.groovy

```diff
@@ -51,27 +52,27 @@ class FilterExperimentDataSpecification extends Specification {
         given:
         HashMap sampleProperties = ["QBiC.Code": "QXXX", "SequencingFacilityId": "IR1234", "otherProp": "value", "Filename": "QXXXIR1234_1.fastq, QXXX.fastq, IR1234_3.fastq"]
         when:
-        boolean res = filterExperimentData.validFilenames(sampleProperties)
+        def res = filterExperimentData.validFilenames(sampleProperties)
         then:
-        res
+        res == null
```

Comment:
I am unsure if I understand this test properly. Why is `null` an expected return value that leads to the acceptance of the test?

Reply:
Valid point. Updated the test by using noExceptionThrown(): d778e77